### DISCUSSION TITLE
Enhancement to /statsedit

### DISCUSSION THREAD

**Discussion by VimaMT** [Zrips/CMI#7824] [Enhancement]
We wish to begin creating mini-events on our servers that are based upon stats such as number of cake slices eaten, etc., during a limited amount of time. To get started with each event we want to clear the specific stat associated with the event for all players on the server.

Can we please add a parameter to /statsedit that will allow specific stats to be set/added/removed for ALL players on the server at one time?

**Comment by MrCrazys** [comment]
You can probably just create a custom alias that uses the `allPlayers!` specialized command. See https://github.com/mrfdev/CMI/blob/master/Resources/Add-ons/globalcommand/1MB-gcmd.yml for an example.

**Reply by VimaMT** [reply]
I'll look at it. Thx.
